src/Navigation/SidePanel/SidePanel.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import SidePanel, { SidePanelProps } from './index.js'
+
+export default {
+  title: 'Components/Navigation/SidePanel',
+  component: SidePanel,
+  argTypes: {
+    heading: {
+      control: 'text',
+    },
+    color: {
+      control: {
+        type: 'select',
+      },
+      options: ['#a7a7a7', '#fff', '#00ff00', '#33e58c', 'red'],
+    },
+  },
+}
+
+const fixtures = [
+  {
+    id: 'heidi',
+    title: 'Heidi Heidi',
+    subtitle: 'The Hydrogen Producer',
+    background: '#AAED93',
+  },
+  {
+    id: 'emma',
+    title: 'Emma Connor',
```

Comment:
**Connor** is the _Consumer_ .. maybe _Emma Energy Prod_ ?

Reply:
Emma if we are talking about how the name of this persona looks like in fool is _Ema the Energy Producer_ i think we should avoid mixing this with the the _Customer_ ( _Connor the Hydrogen Customer_ ).

Source:

![image](https://github.com/digicatapult/ui-component-library/assets/58742260/4d4efced-47bc-4c15-9b35-09879511aaf2)

Reply:
Sorry for mixing a few things up, thought conor will be left outside. This was intended to be purely for references and really should have been some fixtures I think, but this shuold be update now.